frontend/src/router/pages/EditAndReview.jsx

```diff
@@ -1,7 +1,73 @@
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faMagnifyingGlass } from "@fa-kit/icons/classic/solid";
+import { useApiGet } from "@/hooks/useApi";
+import EditAndReviewTable from "@/components/EditAndReviewTable";
+import { useState } from "react";
+import orderBy from "lodash/orderBy";
 
 function EditAndReview() {
+  const { data, loading } = useApiGet("/parks");
+
+  // table filter state
+  const [nameFilter, setNameFilter] = useState("");
+  const [statusFilter, setStatusFilter] = useState("");
+
+  // table sorting state
+  const [sortColumn, setSortColumn] = useState("parkName");
+  const [sortOrder, setSortOrder] = useState("asc");
+
+  function resetFilters() {
+    setNameFilter("");
+    setStatusFilter("");
+  }
+
+  function updateSort(column, order) {
+    setSortColumn(column);
+    setSortOrder(order);
+  }
+
+  // add status values to parks
+  const parks =
+    data?.map((park) => ({
+      ...park,
+      status: `TODO: Active ${park.id}`,
+    })) || [];
+
+  // unique values for filter dropdowns
+  const statusValues =
+    parks
+      .map((park) => park.status)
+      .filter((value, index, self) => self.indexOf(value) === index) || [];
+
+  const filteredParks = parks.filter((park) => {
+    if (
+      nameFilter &&
+      !park.name.toLocaleLowerCase().includes(nameFilter.toLocaleLowerCase())
+    ) {
+      return false;
+    }
+
+    if (statusFilter && park.status !== statusFilter) {
+      return false;
+    }
```

Comment:
From the wireframes it seems the user can select more than one status to filter.

Reply:
💀 I hooked up a crude dropdown/checkbox group with native bootstrap stuff. Ayumi showed me a third party component we can probably use if we need more functionality than that.